doc/classes/Area2D.xml

```diff
@@ -61,7 +61,7 @@
 			<argument index="0" name="body" type="Node">
 			</argument>
 			<description>
-				If [code]true[/code], the given physics body overlaps the Area2D.
+				Returns [code]true[/code] if the given physics body intersects or overlaps Area2D. Otherwise it returns false.
```

Comment:
Same as above:

```suggestion
				Returns [code]true[/code] if the given physics body intersects or overlaps Area2D, [code]false[/code] otherwise.
```